.github/workflows/test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     - name: Lint tinygrad with pylint
       run: python -m pylint tinygrad/
     - name: Run mypy
-      run: python -m mypy
+      run: python -m mypy --warn-unused-ignores
```

Comment:
Can you add this to the mypy config instead?

Reply:
Done, good point!